### ISSUE TITLE
Error parsing XML with attributes starting with an underscore

### ISSUE THREAD

**Issue by drktfl** [prettier/plugin-xml#56]
Hi,

I am trying to run prettier on an XML file like this:

```xml
<?xml version="1.0" encoding="utf-8" ?>
<a _test="5" />
```

When executing `prettier test.xml` (`test.xml` being the file listed above), it fails with:

```
test.xml[error] test.xml: Error: Error parsing XML
[error]     at Object.parse (C:\Temp\node_modules\@prettier\plugin-xml\src\parse.js:7:11)
[error]     at Object.parse (C:\Temp\node_modules\prettier\bin-prettier.js:9743:19)
[error]     at coreFormat (C:\Temp\node_modules\prettier\bin-prettier.js:13258:23)
[error]     at format (C:\Temp\node_modules\prettier\bin-prettier.js:13516:73)
[error]     at formatWithCursor (C:\Temp\node_modules\prettier\bin-prettier.js:13532:12)
[error]     at Object.formatWithCursor (C:\Temp\node_modules\prettier\bin-prettier.js:44213:15)
[error]     at format$1 (C:\Temp\node_modules\prettier\bin-prettier.js:45598:19)
[error]     at C:\Temp\node_modules\prettier\bin-prettier.js:45792:16
[error]     at C:\Temp\node_modules\prettier\bin-prettier.js:45731:14
[error]     at Array.forEach (<anonymous>)
```

This is on prettier 1.19.1 and @prettier/plugin-xml 0.7.1

Thanks and regards,
Dirk

**Comment by kddeisz** [comment]
Fix for this is out in v0.7.2. Thanks for the report!

**Comment by drktfl** [comment]
Thanks for fixing this so quickly!
